service/migration_manager.cc

```diff
@@ -1215,7 +1215,9 @@ future<column_mapping> get_column_mapping(table_id table_id, table_schema_versio
 }
 
 future<> migration_manager::on_join(gms::inet_address endpoint, gms::endpoint_state ep_state) {
-    schedule_schema_pull(endpoint, ep_state);
+    if (_group0_client.using_raft()) {
+        schedule_schema_pull(endpoint, ep_state);
```

Comment:
This is pulling schema if raft is on and not pulling if it's off...

Reply:
Ouch. Fixed in https://github.com/scylladb/scylladb/commit/05ab19bab3995c3db96bd590b8e45915f778f399